docs/.redirects/redirects.json

```diff
@@ -14,8 +20,8 @@
     "architecture/system-architecture": "../get-started/architecture/system-architecture.html",
     "architecture/introduction": "../get-started/architecture/introduction.html",
     "setup-cluster/deploy-cluster/slurm/install-on-slurm": "../../slurm/install-on-slurm.html",
-    "setup-cluster/deploy-cluster/slurm/hpc-with-agent": "../../slurm/_index.html", 
-    "setup-cluster/slurm/hpc-with-agent": "../slurm/_index.html",
+    "setup-cluster/deploy-cluster/slurm/hpc-with-agent": "../../slurm/_index.html",
+    "setup-cluster/slurm/hpc-with-agent": "_index.html",
```

Comment:
Not related, right? ;)

Reply:
I had to fix some redirects using `docs/redirects.py`. This is auto-generated.